app/models/migration/ecf/npq_application.rb

```diff
@@ -0,0 +1,10 @@
+module Migration::Ecf
+  class NpqApplication < BaseRecord
```

Comment:
I wonder if we should pull NPQ application yet, even though we're looking for TRNs those all come from NPQ reeg anyway, maybe we hold off on this for now?